src-lib/image_opencv.cpp

```diff
@@ -236,7 +198,7 @@ cv::Mat image_to_mat(image img)
 			for (int c = 0; c < img.c; ++c)
 			{
 				float val = img.data[c*img.h*img.w + y*img.w + x];
-				mat.data[y*step + x*img.c + c] = (unsigned char)(val * 255);
+				mat.data[y*step + x*img.c + c] = (unsigned char)(val * 255);	 // Is this right?
```

Comment:
This should also have a @todo Doxygen tag so it shows up on the "todo" output page.